src/sentry/api/endpoints/project_dynamic_sampling.py

```diff
@@ -199,11 +207,12 @@ def __fetch_randomly_sampled_root_transactions(
                 "project_id": [project.id],
                 "organization_id": project.organization.id,
             },
-            query=f"{query} event.type:transaction !has:trace.parent_span",
+            query=f"{query} event.type:transaction",
             selected_columns=[
                 "id",
                 "trace",
                 "trace.client_sample_rate",
+                "trace.parent_span",
```

Comment:
What is this used for?

Reply:
oops, it's wrong version. Please try 4d71a03